packages/flutter_tools/templates/application/common/pubspec.yaml.tmpl

```diff
@@ -12,9 +12,14 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+{{#withDriverTest}}
+  flutter_driver:
+    sdk: flutter
+{{/withDriverTest}}
+
 
```

Comment:
I didn't follow whether this is still in or out. But applications need a default dependency on cupertino_icons, otherwise, cupertino apps would break by default.

Reply:
Hmm.  I didn't change anything having to do with cupertino_icons, I don't think.

Reply:
Sorry, my comment was a bit cryptic. What I meant was that the previous default 'app' template had a dependency on `cupertino_icons: ^0.1.2` which was needed for the cupertino icons to show up. It should appear in the module and module+application modes by default as well when re-submitting this.